src/Microsoft.OData.Core/ODataBatchJsonWriter.cs

```diff
@@ -0,0 +1,387 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="ODataBatchJsonWriter.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.OData.Core

```

Comment:
> Microsoft.OData.Core [](start = 9, length = 21)

Classes specific to JSON (light) format should be in Microsoft.OData.Core.JsonLight namespace